test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -93,6 +105,13 @@ class SimpleHttpCacheTest : public testing::Test {
     if (body != actual_body) {
       return AssertionFailure() << "Expected body == " << body << "\n  Actual:  " << actual_body;
     }
+    if (!trailers.empty()) {
```

Comment:
Can you rename this expectLookupSuccessWIthBodyAndTrailers? I also don't think you necessarily want the above if-statement. If the caller doesn't supply trailers we should go ahead and check they are empty, no?

Reply:
Done.